src/EnergyPlus/General.hh

```diff
@@ -97,6 +98,8 @@ namespace General {
         }
     }
 
+    void MovingAvg(EPVector<Real64> &DataIn, int NumItemsInAvg);
+
     void MovingAvg(Array1D<Real64> &DataIn, int NumItemsInAvg);
```

Comment:
@Myoldmopar Some fallout from the consolidation in DataSizing. Some of the common variables were `Array1D` in `ZoneSizingData` but were `EPVector` in `TermUnitZoneSizingData`. `MovingAvg` is only applied to the `ZoneSizingData` side, so the existing function wouldn't accept an `EPVector`. But `MovingAvg` is used on a bunch of other `Array1D`s so there are now two flavors of `MovingAvg`. If you prefer I could remove this and change the affected fields in zone sizing back to `Array1D`.